packages/app/src/settings/reducers/settings.reducer.ts

```diff
@@ -97,24 +96,14 @@ export const settingsReducer = createReducer<SettingsState>(
         state.tethering = action.payload
       })
 
-      .addCase(toggleUpdateAvailable.pending, (state) => {
-        state.loaded = false
-        state.loading = true
-      })
-      .addCase(toggleUpdateAvailable.fulfilled, (state, action) => {
-        state.loaded = true
-        state.loading = false
-        state.updateAvailable = action.payload
+      .addCase(toggleUpdateAvailable.pending, (state, action) => {
```

Comment:
Is it ok that this action is handled in `pending` state, not in `fulfilled`?

Reply:
I think that changing the state during the pending state isn't bad. Maybe, the architecture of the action definitions could be better here but please leave this improvement for another chance.